examples/dataprocserverless-running-notebooks/README.md

```diff
@@ -0,0 +1,100 @@
+Two Step Migration Process for typical customer scenario around this example:  
+
+1. Migrate and stage Spark Notebooks that were in legacy data lake to  GCS then
+2. Set up orchestration for deploying the staged Notebooks on Dataproc Serverless as a Spark batch job 
+
+(This example does not cover the development process, but you can check out more details [here](https://cloud.google.com/vertex-ai/docs/pipelines/dataproc-component) for utilizing [Vertex AI](https://cloud.google.com/vertex-ai) for interactive development)
+
+## File Directory Structure
+
+**dataproc_serverless**
+
+    ├── composer_input                   
+    │   ├── jobs/                       wrapper_papermill.py
+    │   ├── DAGs/                       serverless_airflow.py
+    ├── notebooks 
+    │   ├── datasets/                   electric_vehicle_population.csv
+    │   ├── jupyter/                    spark_notebook.ipynb
+    │   ├── jupyter/output              spark_notebook_outbook.ipynb
+    
+## File Details    
+### composer_input
+* **wrapper_papermill.py**: runs a papermill execution of input notebook and writes the output file into the assgined location
+* **serverless_airflow.py**: orchestrates the workflow 
+  * Dataproc Batch Creation :This operator runs your workloads on Dataproc Serverless
+    ```
+    create_batch1 = DataprocCreateBatchOperator()
+    create_batch2 = DataprocCreateBatchOperator()...
+    ```
+
+### notebooks 
+* **spark_notebook.ibynp** : This file creates Spark Session for Electric Vehicle Population, loads csv file in GCS and perform fundamental Spark functions
+    ```
+    spark = SparkSession \
+        .builder \
+        .appName("Spark Session for Electric Vehicle Population") \
+        .getOrCreate()
+    
+    path=f"gs://{gcs_bucket}/dataproc_serverless/notebooks/datasets/electric_vehicle_population.csv"
+    df = spark.read.csv(path, header=True)
+    df.show()
+    
+    df.filter(df['Model Year'] > 2020).show(5)
+    sortedByElectricRange = df.orderBy('Electric Range').show(10)
+    ```
+* **jupyter/output** : this is where papermill notebook execution outputs will be stored
+        Sample Notebook Output
+        ![Screenshot 2023-03-16 at 2 20 35 AM](https://user-images.githubusercontent.com/123537947/225531950-dccde663-fade-413d-9600-eb59ab669800.png)
+
+* **electric_vehicle_population.csv** : this truncated [dataset](https://catalog.data.gov/dataset/electric-vehicle-population-data) shows the Battery Electric Vehicles (BEVs) and Plug-in Hybrid Electric Vehicles (PHEVs) that are currently registered through Washington State Department of Licensing (DOL).
+
+# I) Orchestrating End to End Notebook Execution workflow in Cloud Composer 
+
+1. Make sure to modify gcs path for datasets in Notebook 
+
+2. Create [Persistent History Server](https://cloud.google.com/dataproc/docs/concepts/jobs/history-server)
```

Comment:
Wouldn't this be optional? Why did you set it as a necessary step? Removing this step and pointing out the documention if the user wants to use PHS would increase simplicity and possibly adoption